docs/updating.rst

```diff
@@ -2,35 +2,55 @@
 How to update the Atomic Developer Bundle to the latest version
 ===============================================================
 
-* If you are using the image from atlas.hashicorp.com
+Depending on your implementation of the ADB, you might be able to directly
+update your Vagrant box. In some cases, you will need to perform a clean
+installation or follow additional steps to complete the update.
```

Comment:
@thatdocslady @bexelbie :
Yes clean installation requires `vagrant destroy` and we should add a warning that this step will potentially remove any data or configuration the user performed with current ADB VM and it is the user's decision whether to update the Vagrant box or not.